### ISSUE TITLE
check of .Rproj for linters yields false positive

### ISSUE THREAD

**Issue by hansvancalster** [inbo/checklist#50]
I have the following .Rproj file:

```
Version: 1.0

RestoreWorkspace: Default
SaveWorkspace: Default
AlwaysSaveHistory: Default

EnableCodeIndexing: Yes
UseSpacesForTab: Yes
NumSpacesForTab: 2
Encoding: UTF-8

RnwWeave: knitr
LaTeX: XeLaTeX

AutoAppendNewline: Yes

```

which is flagged by `checklist::check_source()`:

![image](https://user-images.githubusercontent.com/18713718/117833543-b5bed680-b276-11eb-9f20-2c8b344d0db0.png)

so probably something needs to be changed so that the .Rproj file is excluded from the lintr check.

**Comment by hansvancalster** [comment]
Closed by #51
